jdat_notebooks/niriss_ami_binary/1_niriss_ami_binary.ipynb

```diff
@@ -0,0 +1,645 @@
+    "  82.18740518  -65.44767541   5.88850   5.49770   5.07560   4.95800   4.83650   4.72940   4.72220   4.61000   4.61000   4.61000   4.61000   4.61000\n",
+    "  82.18717120  -65.44764863  12.06621  10.73581  10.47740  10.15193   9.84442   9.76398   9.75940   8.99121   8.69367   8.76689   8.81310   8.81310\n",
+    "  82.18534722  -65.44612065   8.88930   8.38010   7.86780   7.70440   7.54010   7.38740   7.38650   7.29090   7.28470   7.33890   7.38820   7.49960\n",
+    "  82.19501500  -65.44532800  12.13360  12.89690  12.47030  12.40400  12.28660  13.19490  14.01950  14.60890  14.67080  15.09080  15.20730  15.56880\n",
+    "  82.19343600  -65.45299500  13.54130  13.63920  13.68400  13.69520  13.70530  13.74310  13.76640  13.78850  13.78900  13.79520  13.79330  13.79180\n",
+    "  \n",
+    "...more sources...\n",
+    "\n",
+    "# \n",
+    "# vegamag\n",
+    "# \n",
+    "# \n",
+    "#x_or_RA y_or_Dec niriss_f090w_magnitude niriss_f115w_magnitude niriss_f140m_magnitude niriss_f150w_magnitude niriss_f158m_magnitude niriss_f200w_magnitude niriss_f277w_magnitude niriss_f356w_magnitude niriss_f380m_magnitude niriss_f430m_magnitude niriss_f444w_magnitude niriss_f480m_magnitude\n",
+    "  \t\n",
+    "  82.78450088  -65.12833088   7.33570   6.74320   6.18410   5.99130   5.80110   5.61850   5.62170   5.49700   5.49700   5.53100   5.53100   5.53100\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "catalogues = {'AB-DOR': {'point_source':  './mirage_input_files/stars_field19_20_combined_allfilters_new.list'\n",
+    "                        },\n",
+    "              'HD-37093': {'point_source': './mirage_input_files/stars_field19_20_combined_allfilters_new.list'\n",
+    "                        }\n",
+    "             }   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set the telescope roll angle PAV3 for each observation. Another way to get PAV3 is from the APT file Reports."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obs1 = 1\n",
+    "pav3_obs1 = yaml_generator.default_obs_v3pa_on_date(pointing_name, obs1, date='2022-04-01')\n",
+    "obs2 = 2\n",
+    "pav3_obs2 = yaml_generator.default_obs_v3pa_on_date(pointing_name, obs2, date='2022-04-01')\n",
+    "\n",
+    "roll_angle = {'001':pav3_obs1 , '002': pav3_obs2}\n",
+    "\n",
+    "dates = '2022-04-01'\n",
+    "reffile_defaults = 'crds'\n",
+    "datatype = 'raw'\n",
+    "print(\"PAV3 for observation 1\", pav3_obs1, \"PAV3 for observation 1\", pav3_obs2)"
```

Comment:
The second instance of "PAV3 for observation 1" should be changed to "PAV3 for observation 2"